mappings/net/minecraft/block/Block.mapping

```diff
@@ -450,7 +450,7 @@ CLASS bgs net/minecraft/block/Block
 	METHOD n getFrictionCoefficient ()F
 	METHOD n hasRandomTicks (Lbpm;)Z
 		ARG 1 state
-	METHOD n (Lbpm;Lbbb;Let;)Z
+	METHOD n isFullCube (Lbpm;Lbbb;Let;)Z
```

Comment:
Can you clarify the difference between `isFullCube` and the two `isShapeFullCube` methods?

Reply:
`isShapeFullCube` operates on a `VoxelShape` - it basically checks if it's equivalent to `VoxelShapes.fullCube()`. `isFullCube` accepts a world, block state and block position, gets the collision box and then uses `isFullCube`.

Reply:
I think we should go with this for now and see what happens. I'm all for naming Block methods based on *uses*, not *behavior*, but that's tricky.